src/stories/Library/recommender/recommender.scss

```diff
@@ -0,0 +1,82 @@
+.recommender {
+  background-color: $c-global-tertiary-2;
+
+  @include breakpoint-m {
+    padding: $s-6xl;
+  }
+}
+
+.recommender__title {
+  color: $c-text-primary-white;
+  text-align: center;
+  padding: $s-lg;
+
+  @include breakpoint-m {
+    padding-bottom: $s-6xl;
+  }
+}
+
+.recommender__grid {
+  display: grid;
+  grid-template-columns: 50% 50%;
+  grid-template-rows: 50% 50%;
+
+  @include breakpoint-m {
+    grid-template-columns: 33.33% 33.33% 33.33%;
+  }
+}
+
+.recommender-material {
+  border: 1px solid $c-text-secondary-gray;
+  padding: 30px;
+  display: flex;
+  width: 100%;
+  position: relative;
+  height: 350px;
+
+  @include breakpoint-m {
+    &:last-of-type {
+      display: none;
+    }
+  }
+
+  .recommender-material__favourite {
+    position: absolute;
+    top: 10px;
+    right: 10px;
+  }
+
+  .recommender-material__cover-container {
+    display: flex;
+    justify-content: center;
+    width: 100%;
+    margin-top: 10px;
+  }
+
+  .recommender-material__meta {
+    position: absolute;
+    left: 10px;
+    bottom: 10px;
+    width: 100%;
+    padding-right: 10px;
+  }
+
+  .recommender-material__meta__title {
```

Comment:
Please rework this class name.

It has multiple elements within a block. I suggest `.recommender-material__title`. You are allowed to nest elements within a block as long as the block constitutes a semantic element.